main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/ministryofjustice/analytical-platform-auth0-user-exports/auth"
+	"log"
+	"os"
+	"time"
+
+	session2 "github.com/aws/aws-sdk-go/aws/session"
+	env "github.com/ministryofjustice/analytical-platform-auth0-user-exports/default_env_value"
+	connection "github.com/ministryofjustice/analytical-platform-auth0-user-exports/get_connection"
+	"github.com/ministryofjustice/analytical-platform-auth0-user-exports/job"
+)
+
+var (
```

Comment:
can these be constants or need to change? (genuine question). I know that some of these values are env-dependant but once the app runs most of them probably don't need to change (and it's safer to set to constant unless they need to change)

Reply:
Actually, slightly confused.  Can an env variable be a constant? I'll google when to use constants in non-obvious situations because I honestly don't know 🤷‍♂

Reply:
good point, you're probably right because `const` would need to be there at compile time, ignore me ☺️